code/modules/antagonists/brainwashing/brainwashing.dm

```diff
@@ -41,9 +43,14 @@
 	data["objectives"] = get_objectives()
 	return data
 
+/datum/antagonist/brainwashed/greet()
+	. = ..()
+	add_antag_hud(antag_hud_type, antag_hud_name, owner.current)
+
 /datum/antagonist/brainwashed/farewell()
 	to_chat(owner, span_warning("Your mind suddenly clears..."))
 	to_chat(owner, "<big>[span_warning("<b>You feel the weight of the Directives disappear! You no longer have to obey them.</b>")]</big>")
+	remove_antag_hud(antag_hud_type, owner.current)
```

Comment:
You should use apply_innate_effects and remove_innate_effects as those get called when the antagonist also transfers bodies.

Reply:
Fixed!